.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -15,6 +15,14 @@ See https://github.com/ampproject/amphtml/blob/master/contributing/contributing-
 If you haven't already done so, sign the Contributor License Agreement (CLA) as soon as possible to avoid delays merging your code. A signed CLA is not necessary to submit this I2I or to send a pull request, but it will be needed before your code can be merged. See https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#contributor-license-agreement for more information on CLAs.
 -->
 
+## Status
+- [ ] Draft
+- [ ] Ready for review
+
+<!--
+Issues that aren't marked "Ready for review" are considered drafts. Draft I2Is are encouraged for complex or cross functional proposals in order to promote broader community awareness and get earlier feedback.
```

Comment:
Not sure if "cross functional" will be understood. How about some wording like "... are encouraged for complex proposals and/or those expected to impact many stakeholders or parts of the project in order to promote ..."